core-web/apps/dotcms-ui/src/app/app-routing.module.ts

```diff
@@ -60,6 +60,17 @@ const PORTLETS_ANGULAR: Route[] = [
                 (m) => m.DotContentTypesModule
             )
     },
+    {
+        path: 'languages',
+
+        canActivate: [MenuGuardService],
+        canActivateChild: [MenuGuardService],
+        data: {
+            reuseRoute: false
+        },
```

Comment:
What is this for?

Reply:
we implement a custom `RouteReuseStrategy` this indicate that if the params change in the same route, do not re-use the route, instead load all again.